rails-commenting.rb

```diff
@@ -5,24 +5,25 @@
 
 # app/controller/blog_posts_controller.rb
 
-# 1)
+# 1) rails g controller
 class BlogPostsController < ApplicationController
   def index
     # 2)
     @posts = BlogPost.all
   end
+# instance variable that contains all objects in BlogPost
 
   def show
-    # 3)
+    # 3)method that displays one id
     @post = BlogPost.find(params[:id])
   end
 
-  # 4)
+  # 4)method used to post a new obj
   def new
   end
 
   def create
-    # 5)
+    # 5)method used to post a new obj
```

Comment:
yes! and creates an instance variable that will create a new blog post that will check for the params being satisfied, will redirect if valid